crates/hive-utils/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # TODO: Use official madara image when https://github.com/keep-starknet-strange/madara/issues/982 is closed
 # FROM ghcr.io/keep-starknet-strange/madara:v0.1.0.experimental.3 as madara
-FROM fredtupas/madara as madara
+FROM fredtupas/madara-fixed as madara
```

Comment:
TODO: will update pr to official image once this pr in Madara is merged https://github.com/keep-starknet-strange/madara/pull/1018